README.md

```diff
@@ -834,12 +917,6 @@ OK:
 TODO: - no Errors/warnings during the build
 TODO-A: - no incautious use of malloc/sprintf (as far as we can check it)
 TODO-B: - no incautious use of malloc/sprintf (the language has no direct MM)
-TODO: - no use of sudo, gksu, pkexec, or LD_LIBRARY_PATH (usage is OK inside
-TODO:   tests)
-TODO: - no use of user nobody
-TODO: - no use of setuid
-TODO: - use of setuid, but ok because TBD (prefer systemd to set those
-TODO:   for services)
 TODO: - no important open bugs (crashers, etc) in Debian or Ubuntu
```

Comment:
Reviewing bug history may show neglected packages, or packages that are too costly to maintain, etc. Maybe it needs its own bullet point, though?

Reply:
We already have marked the checks of regular updates in upstream and in packaging as "to be looked at" for the re-review. But your thought is good. I do not think we need another check for it, but I added a hint right above the check what might be special on a re-review.